test/config/credentials_default.json

```diff
@@ -19,8 +20,8 @@
     ],
     "ora": [
       {
-        "username": "vagrant",
-        "password": "vagrant"
+        "username": "onrack",
+        "password": "onrack"
```

Comment:
Also:  Our default community is "onrack".  Is that what we want?  Do we want "public" or something else to appear out on github?